olive/passes/onnx/vitis_ai/calibrate.py

```diff
@@ -59,6 +59,10 @@ def augment_graph(self):
         Make all quantization_candidates op type nodes as part of the graph output.
         :return: augmented ONNX model
         """
+        # not supported in ORT >= 1.16.0
+        # TODO: Update code to support different versions of ORT
+        from onnxruntime.quantization.quant_utils import clone_model_with_shape_infer
```

Comment:
should we add try/except for ImportError? if the exception happens, it seems we can directly use the model without clone? @sheng-xiao

Reply:
Maybe we can do it in a separate PR? I have created a TODO here for tracking.

Reply:
sure

Reply:
Thanks!